### PR TITLE
add documentation for POST /logs

### PR DESCRIPTION
**Pull request by cpanato**
PR: https://github.com/mattermost/platform/pull/6143

### CODE REVIEW

File: v4/source/system.yaml [hunk 1]
```diff
@@ -230,6 +230,40 @@
         '403':
           $ref: '#/responses/Forbidden'
 
+    post:
+      tags:
+        - system
+      summary: Post logs
```
Comment (jwilander) [inline v4/source/system.yaml]: Can we update this to "Add log message"?

File: v4/source/system.yaml [hunk 2]
```diff
@@ -230,6 +230,40 @@
...
+      description: |
+        Post logs to the server logs.
```
Comment (jwilander) [inline v4/source/system.yaml]: And this to "Add log messages to the server logs."?

File: v4/source/system.yaml [hunk 3]
```diff
@@ -230,6 +230,40 @@
...
+        ##### Permissions
+        Must have `manage_system` permission.
```
Comment (jwilander) [inline v4/source/system.yaml]: And here maybe mention that if `ServiceSettings.EnableDeveloper` is set to true in the config file, then any user can use this endpoint

File: v4/source/system.yaml (hunk 1 above)
Comment (cpanato) [inline v4/source/system.yaml]: done

File: v4/source/system.yaml (hunk 2 above)
Comment (cpanato) [inline v4/source/system.yaml]: done

File: v4/source/system.yaml (hunk 3 above)
Comment (cpanato) [inline v4/source/system.yaml]: done.

**Pull request merged**